### DISCUSSION TITLE
Please, please, add autowall!

### DISCUSSION THREAD

**Discussion by diekunstderfuge** [danielkrupinski/Osiris#3428] [Ideas]
This small addition would make a BIG difference in this awesome hack.

**Comment by Nep-Timeline** [comment]
?

**Reply by Frouk3** [reply]
> ?

its a linux issue

**Reply by Nep-Timeline** [reply]
oh, #3344

**Comment by diekunstderfuge** [comment]
**Killshot** was disabling **autowall** in all weapons except the sniper rifles.

I still don't understand what killshot does, I've read many people trying to guess it with a lot of conflicting guesses.

Nevertheless, if you are having the same problem that I was, try disabling killshot.
